### DISCUSSION TITLE
Configuring HLS Playback Permissions Without JWT Authentication for Livestreams

### DISCUSSION THREAD

**Discussion by hunglsxx** [bluenviron/mediamtx#3495] [Questions]
### Question

Hello,

I am using JWT authentication with the ```authMethod: jwt``` configuration. I have followed the instructions and used Keycloak, and everything seems to work fine when I livestream with RTMP. 
However, with JWT, how can I configure playback permissions using HLS to allow any user to view the stream? I do not want the client side to have to authenticate with JWT; in other words, I do not want to restrict the viewing rights of my livestream.

**Comment by aler9** [comment]
The feature you're looking for is the `authJWTExclude` setting, and is being developed here: #3431

[a comment by an automated account is not shown]
